app/components/Home.js

```diff
@@ -60,6 +60,14 @@ class Home extends Component {
       }
     );
 
+    const profile = localStore.get(availableSettings.profile);
```

Comment:
No need for you to do this. This is only needed if the homepage needs this value from the state